### ISSUE TITLE
Is gvm abandoned ?

### ISSUE THREAD

**Issue by slayer** [moovweb/gvm#330]
Hi @voxxit 

Based on last commits and many unreplied issues it looks like that gvm is abandoned, is it?
If yes, can you recommend alternative?

Thanks

**Comment by brianonn** [comment]
@slayer Are you having troubles ? you can always fork it and fix it.   There are some problems with it that I have easily fixed myself. I just fixed my local installed copy and did not fork it though.

**Comment by slayer** [comment]
Hi @brianonn 
Before start using (and continue using) some tool usually I check activity of project - recent commit history, open issues, PRs, etc to evaluate how it "alive".
I use gvm several years and week ago I installed it on fresh system (and fixed it for my own needs).

As you can see there are no recent commits, few stalled PRs and to much open issues.
Looks like author abandoned the project. There are many cases in OSS when contributors try to improve projects but their efforts remain in vain.
So my aim is evaluate how alive this project is.

**Comment by bionicles** [comment]
i literally just used gvm to successfully install go1.13.4 (latest version) on ubuntu 16.04 as of 26 november 2019 ... recent activity or not, it works

**Comment by brianonn** [comment]
@bionicles @slayer it does work except for it's handling of aliases.  I have an extensive alias list for `ls` `grep` `sed` etc... and by default it uses your aliases and gets unexpected failures.  I've fixed a few of these myself when I came across them in my own installation. 

This issue (below) in particular needs to be patched in any installation if you have any aliases for the above tools.  

https://github.com/moovweb/gvm/issues/234
